src/Sentry.Unity.Editor/Native/BuildPostProcess.cs

```diff
@@ -75,29 +75,32 @@ public static void OnPostProcessBuild(BuildTarget target, string executablePath)
     private static void AddCrashHandler(IDiagnosticLogger logger, BuildTarget target, string buildOutputDir, string executableName)
     {
         string crashpadPath;
-        if (target is BuildTarget.StandaloneWindows64)
-        {
-            crashpadPath = Path.Combine("Windows", "Sentry", "crashpad_handler.exe");
-        }
-        else if (target is BuildTarget.StandaloneLinux64)
-        {
-            // No standalone crash handler for Linux - uses built-in breakpad.
-            return;
-        }
-        else if (target is BuildTarget.StandaloneOSX)
-        {
-            // No standalone crash handler for macOS - uses built-in handler in sentry-cocoa.
-            return;
-        }
-        else
+        string werHandlerPath;
+
+        switch (target)
         {
-            throw new ArgumentException($"Unsupported build target: {target}");
+            case BuildTarget.StandaloneWindows64:
+                crashpadPath = Path.Combine("Windows", "Sentry", "crashpad_handler.exe");
+                werHandlerPath = Path.Combine("Windows", "Sentry", "crashpad_wer.dll");
+                break;
+            case BuildTarget.StandaloneLinux64:
+            case BuildTarget.StandaloneOSX:
+                // No standalone crash handler for Linux/macOS - uses built-in handlers.
+                return;
+            default:
+                throw new ArgumentException($"Unsupported build target: {target}");
         }
 
-        crashpadPath = Path.GetFullPath(Path.Combine("Packages", SentryPackageInfo.GetName(), "Plugins", crashpadPath));
-        var targetPath = Path.Combine(buildOutputDir, Path.GetFileName(crashpadPath));
-        logger.LogInfo("Copying the native crash handler '{0}' to {1}", Path.GetFileName(crashpadPath), targetPath);
-        File.Copy(crashpadPath, targetPath, true);
+        CopyHandler(logger, buildOutputDir, crashpadPath);
+        CopyHandler(logger, buildOutputDir, werHandlerPath);
+    }
```

Comment:
maybe move `CopyHandler()` calls to the windows switch case? No need for predeclared variables